@xen-orchestra/web/src/composables/auth.composable.ts

```diff
@@ -0,0 +1,23 @@
+import { useCookies } from '@vueuse/integrations/useCookies'
+
+export const useAuth = () => {
+  const { get, remove } = useCookies()
+
+  const logout = () => {
+    if (get('token') === undefined) {
+      console.error('No "token" cookie found')
+
+      return
+    }
+
+    remove('token')
+
+    window.location.href = import.meta.env.DEV
+      ? `${import.meta.env.VITE_XO_REST_BASE_URL}/signin#/`
+      : `${window.origin}/signin#/`
```

Comment:
I'm not sure this should be handled here.

I think the redirect to the login page should be handled by the page which is being accessed.

For example, if we are logging out from a `/foo/bar` page, and that page needs to be logged in... then a router hook should do the redirect (possibly with something like `?from=/foo/bar` in the future).